components/gcp/bigquery/query/component.yaml

```diff
@@ -16,6 +16,9 @@ name: Bigquery - Query
 description: |
   A Kubeflow Pipeline component to submit a query to Google Cloud Bigquery 
   service and dump outputs to a Google Cloud Storage blob. 
+metadata:
+  labels:
```

Comment:
Is it possible to use annotation instead of label?

Reply:
I have replied in another comment and you probably missed that. Label is better to serve as selector instead of annotation. 

https://kubernetes.io/docs/concepts/overview/working-with-objects/labels/#equality-based-requirement